python/README.md

```diff
@@ -24,7 +24,7 @@ able to edit the source files.
 
 For convenience, you may run the following commands from this directory:
 
-    $ make          # Deploy the module in "development mode"
+    $ make          # Build the module in-place
```

Comment:
Yeah, that sounds good to me, though perhaps we should remove the references to "development mode" in the README if the in-place build is the default.